compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below, we'll let you know when your place is confirmed. Sign-ups close 19th September 2020.
```

Comment:
```suggestion
          Please register your interest below, we'll let you know when your place is confirmed. Sign-ups close 19th November 2020.
```

Don't think time travel should be a prerequisite for competing this year.